Apps/W1/Shopify/app/src/Logs/Pages/ShpfySkippedRecords.Page.al

```diff
@@ -0,0 +1,61 @@
+namespace Microsoft.Integration.Shopify;
+
+/// <summary>
+/// Page Shpfy Skipped Records (ID 30166).
+/// </summary>
+page 30166 "Shpfy Skipped Records"
```

Comment:
We should probably add these actions to this page:
![image](https://github.com/user-attachments/assets/30214ed1-d126-4c8d-ad43-eaa733b1da3c)

@AndreiPanko please state if otherwise

Reply:
Agree. Should be aligned